app/models/embeddable.rb

```diff
@@ -34,7 +33,7 @@ def self.table_name_prefix
   end
 
   def self.valid_type(class_string)
-    return Types.detect{|k,v| k.to_s == class_string }
+    return class_string == Embeddable::EmbeddablePlugin.to_s || Types.detect{|k,v| k.to_s == class_string }
```

Comment:
I am not sure I understand why this was moved from line 10. If line 10 was  changed to:

`Embeddable::EmbeddablePlugin => Embeddable::EmbeddablePlugin.to_s` 

It would be  the same  right?

Reply:
I wanted to remove the generic plugin option from the embeddable dropdown so I have to remove it from that list.